1.10/deploying-services/expose-service.md

```diff
@@ -43,35 +52,35 @@ To launch a service on a public node, you must create a Marathon app definition
     ```
 
     If this is added successfully, there is no output.
```

Comment:
See line 48 above:

Add the your app to Marathon by using this command, where `myApp.json` is your app:

This has an extra word at the beginning and could be more precise. Suggest:

Add your app to Marathon by using this command, where `myApp.json` is the file containing your Marathon app definition.